### ISSUE TITLE
model inception_v3 didn't work

### ISSUE THREAD

**Issue by Wwwwei** [cameronfabbri/Compute-Features#2]
hello, I run the model inception_v3 for test to get features, but it didn't work. Is there any bug?

Tensor name "InceptionV3/AuxLogits/Conv2d_2a_3x3/BatchNorm/beta" not found in checkpoint files ./feature-models/inception_v3.ckpt
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
	 [[Node: save/RestoreV2/_301 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_306_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

**Comment by cameronfabbri** [comment]
Hi. There isn't really a bug, just for some reason some of the checkpoints that Tensorflow provided do not match up with their code in the `nets/` folder. You'll just have to use another model (Inception V1 works just fine)

**Comment by Wwwwei** [comment]
ok, I'll try another model. Thanks.
